### PR TITLE
Certificate generation for participants

### PR DESCRIPTION
**Pull request by gldraphael**
- [x] HTML to PDF using NodeServices ~(need to configure the PDF correctly)~
- [x] Certificate template
- [x] Certificate model and migrations
- [x] View Certificate Page
- [x] Download certificate 
- [x] Generate and email certificates

Related Issue: #55

### CODE REVIEW

File: src/EventManagement.Web/ViewModels/Templates/Certificates/CourseCertificateVM.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace losol.EventManagement.Web.ViewModels.Templates.Certificates
+{
+    public class CourseCertificateVM
```
Comment (losviko) [inline src/EventManagement.Web/ViewModels/Templates/Certificates/CourseCertificateVM.cs]: Consider renaming to CertificateVM

Comment (gldraphael) [reply src/EventManagement.Web/ViewModels/Templates/Certificates/CourseCertificateVM.cs]: Done in cdb0b34

File: src/EventManagement.Domain/CourseCertificate.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public Guid CertificateGuid { get; set; }
+
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public Guid AuthCode { get; set; }
+
+        [Required]
+        public string Title { get; set; }
+
+        [Required]
```
Comment (losviko) [inline src/EventManagement.Domain/CourseCertificate.cs]: Remove required here?

File: src/EventManagement.Domain/CourseCertificate.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using losol.EventManagement.Domain;
+
+namespace losol.EventManagement.Domain
+{
+    public class CourseCertificate
```
Comment (losviko) [inline src/EventManagement.Domain/CourseCertificate.cs]: Rename to Certificate?

File: src/EventManagement.Domain/CourseCertificate.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        [Key]
+        public int CourseCertificateId { get; set; }
```
Comment (losviko) [inline src/EventManagement.Domain/CourseCertificate.cs]: Rename to CertificateId?

**Pull request merged**